go_test.go

```diff
@@ -143,12 +145,39 @@ func TestGoWithGlobalConfig(t *testing.T) {
 
 	prepareGoProject(newHomeDir, t, false)
 	runGo(ModuleNameJFrogTest, tests.GoBuildName, buildNumber, t, "go", "build", "--build-name="+tests.GoBuildName, "--build-number="+buildNumber, "--module="+ModuleNameJFrogTest)
-
 	assert.NoError(t, os.Chdir(wd))
 
 	cleanGoTest()
 }
 
+func TestGoGetSpecificVersion(t *testing.T) {
+	// Init test and preper Global config
+	initGoTest(t)
+	buildNumber := "1"
+	oldHomeDir, newHomeDir := prepareHomeDir(t)
+	defer os.Setenv(coreutils.HomeDir, oldHomeDir)
+	defer os.RemoveAll(newHomeDir)
+	wd, err := os.Getwd()
+	assert.NoError(t, err)
+	prepareGoProject("", t, true)
+	// Build and publish a go project
+	runGo("", tests.GoBuildName, buildNumber, t, "go", "build", "--build-name="+tests.GoBuildName, "--build-number="+buildNumber)
+
+	// Act
+	// Go get one of the known pushed dependencies
+	err = tests.NewJfrogCli(execMain, "jfrog rt", "").Exec("go", "get", "rsc.io/quote@v1.5.2", "--build-name="+tests.GoBuildName, "--build-number="+buildNumber)
+	assert.NoError(t, err)
+	artifactoryCli.Exec("bp", tests.GoBuildName, buildNumber)
+	buildInfo, _ := inttestutils.GetBuildInfo(artifactoryDetails.Url, tests.GoBuildName, buildNumber, t, artHttpDetails)
+
+	//Assert
```

Comment:
Let's remove the "//Assert" comment, because currently it is currently not part of our testing code conventions.